pkg/extensions/config/config.go

```diff
@@ -12,11 +12,16 @@ type BaseConfig struct {
 }
 
 type ExtensionConfig struct {
-	Search  *SearchConfig
-	Sync    *sync.Config
-	Metrics *MetricsConfig
-	Scrub   *ScrubConfig
-	Lint    *LintConfig
+	Search    *SearchConfig
+	Sync      *sync.Config
+	Metrics   *MetricsConfig
+	Scrub     *ScrubConfig
+	Lint      *LintConfig
+	AuthQuery *AuthQueryConfig
```

Comment:
Let's call this Mgmt instead, so that we can return/manage config from here

Reply:
but in this case we return only the authentication type
should we call it mgmt/auth ?

Reply:
no, mgmt/config and we return only parts based on access policy